runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilitySession.java

```diff
@@ -68,4 +71,19 @@ void resetLapCount()
 		totalLaps = 0;
 		lapsTillLevel = 0;
 	}
+
+	int getLapsHr()
```

Comment:
I feel like there's no need for three separate methods and reduces readability in this case.